rotkehlchen/db/history_events.py

```diff
@@ -397,39 +401,55 @@ def get_history_events(
         TODO: To not query all columns with all joins for all cases, we perhaps can
         peek on the entry type of the filter and adjust the SELECT fields accordingly?
         """
+        bindings, type_idx = [], 0
         base_prefix = 'SELECT '
```

Comment:
base prefix is always select right? Why not use it always? THat avoids replacing that is more expensive